eahub/feedback/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+## added by Lisa
+class Feedback(models.Model):
+    email = models.CharField(blank=True, max_length=30)
+    message = models.CharField(max_length=2000)
+    page_url = models.CharField(blank=True, max_length=50)
```

Comment:
50 is too low. Max length varies with browser, I think we should go at least with 2048 (yes, we probably will never have urls like that, but we don't have to be concerned about data volume here): https://stackoverflow.com/a/33733386